jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/network/PerThreadNetworkWriteHandler.java

```diff
@@ -0,0 +1,59 @@
+package io.opentelemetry.contrib.jfr.metrics.network;
+
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.metrics.BoundDoubleHistogram;
+import io.opentelemetry.api.metrics.Meter;
+import io.opentelemetry.contrib.jfr.Constants;
+import io.opentelemetry.contrib.jfr.metrics.RecordedEventHandler;
+import jdk.jfr.consumer.RecordedEvent;
+
+public class PerThreadNetworkWriteHandler implements RecordedEventHandler {
+  public static final String SIMPLE_CLASS_NAME =
+      PerThreadNetworkWriteHandler.class.getSimpleName();
+  public static final String BYTES_WRITTEN = "bytesWritten";
+  public static final String THREAD_NAME = "thread.name";
+  public static final String JFR_SOCKET_WRITE_BYTES_WRITTEN = "jfr.SocketWrite.bytesWritten";
```

Comment:
Same as for reads:

Suggestion: `runtime.jvm.network.io` with attribute denoting "write"

For duration, `runtime.jvm.network.duration` with attribute denoting "write"